br/pkg/restore/split/client.go

```diff
@@ -628,7 +628,7 @@ func (c *pdClient) SetStoresLabel(
 	return nil
 }
 
-func (c *pdClient) getPDAPIAddr() string {
+func (c *pdClient) getpdhttpAddr() string {
```

Comment:
name it `getPDHTTPAddr`?

Reply:
I think I replaced it globally mistakenly. Changed back. PTAL.